__tests__/app.test.js

```diff
@@ -0,0 +1,24 @@
+const app = require("../app")
+const query = require("supertest")
+const seed = require("../db/seeds/seed")
+const db = require('../db/connection');
+const data = require('../db/data/test-data/index')
+
+beforeEach(()=>{
+   return seed(data);
+})
+afterAll(()=>{
+    db.end();
+})
+
+describe("app!!", ()=>{
+    test('should return all the topics', ()=>{
+        return query(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response)=>{
+            const {topic} = response.body
+            return response.body
```

Comment:
We don't have any expect statements here so we're not actually testing the response which we are getting back.
lets compare the topic which we have got back and expect it to look how we are expecting.